rllib/evaluation/worker_set.py

```diff
@@ -599,8 +599,22 @@ def valid_module(class_path):
             )
         # JSON file or list of JSON files -> Use JsonReader (shuffled).
         else:
-            input_creator = lambda ioctx: ShuffledInput(
-                JsonReader(config["input"], ioctx), config["shuffle_buffer_size"]
+            _input = config["input"]
```

Comment:
will it be easier if we just adapt the input into dataset reader format, so we can simply reuse the ``elif config["input"] == "dataset":`` block above?
and then we can just throw error msg here saying "unrecognized input format if we reach here".